src/meta/src/barrier/info.rs

```diff
@@ -96,20 +86,6 @@ impl InflightActorInfo {
             map
         };
 
-        let actor_map_to_send = {
-            let mut map: HashMap<_, HashSet<_>> = HashMap::new();
-            for info in actor_infos
-                .fragment_infos
-                .values()
-                .filter(|info| info.is_injectable)
```

Comment:
Why do we no longer need this?

Reply:
We no longer maintain the `actor_map_to_send` in the `InflightActorInfo`. When `inject_barrier`, we will calculate the `actor_ids_to_send` by the providing fragment info.